### DISCUSSION TITLE
2.2.0

### DISCUSSION THREAD

**Discussion by ZappyMan** [ZappyMan/MorphoMetriX#15] [Announcements]
Addressed issues with PySide 6.5.1 and MacOS 10.X, downgraded PySide6 dependency to 6.4.3 for extended MacOS support.

Updated features:
- Added Morphometrix version # to crash log
- Added option to change reticle shape from crosshair to dot
- Added opacity control for reticles
- Fixed issue with exit button not exiting application
- Added value checking for input boxes

***Windows Version Coming January 3rd 2024***

<hr /><em>This discussion was created from the release <a href='https://github.com/ZappyMan/MorphoMetriX-V2/releases/tag/v2.2.0'>2.2.0</a>.</em>

**Comment by melissacook-noaa** [comment]
Is it possible for you to add a certificate to the executable file? I work for the government and our work laptops will check for that information. Since it isn't there, I am not able to download it. We have tried everything and my IT person said to contact you and ask about the certificate. Thanks

**Reply by ZappyMan** [reply]
Hi Melissa,

Our software is (currently) uncertified as it requires annual fees. We're in the process of securing funding and hope to have our application registered by the end of this year. Until then, an alternative is to run the Python version of our application. I'll add more detailed instructions on installation and running this version by the end of today. Hope this helps!

**Comment by HBadSL** [comment]
Hello there,

I've been trying to open Morphometrix by double-clicking on the icon, and it just does not open not matter how many times I try.
